api/lib/reducers/cosmosV0-reducers.js

```diff
@@ -135,18 +135,18 @@ function tallyReducer(proposal, tally, totalBondedTokens) {
   }
```

Comment:
this is cosmosV0 - does this need to be added to V2 as well?

Reply:
no